docs/overview/install.md

```diff
@@ -14,20 +14,14 @@ brew install sqlc
 sudo snap install sqlc
 ```
 
-## go install 
+## go install
 
-### Go >= 1.17:
+Installing recent versions of sqlc requires Go 1.21+.
 
 ```
 go install github.com/sqlc-dev/sqlc/cmd/sqlc@latest
 ```
 
-### Go < 1.17:
-
-```
-go get github.com/sqlc-dev/sqlc/cmd/sqlc
-```
-
```

Comment:
I removed this since it's irrelevant now that Go 1.21 is required for maps and slog.